diff --git a/node_modules/prompts/lib/util/action.js b/node_modules/prompts/lib/util/action.js
index fefbd94..6ae743e 100644
--- a/node_modules/prompts/lib/util/action.js
+++ b/node_modules/prompts/lib/util/action.js
@@ -9,6 +10,8 @@ module.exports = (key, isSelect) => {
     if (key.name === 'd') return 'abort';
     if (key.name === 'e') return 'last';
     if (key.name === 'g') return 'reset';
+    if (key.name === 'p') return 'up';
+    if (key.name === 'n') return 'down';
   }
   
   if (isSelect) {
